orchestrator/graphql/utils/is_query_detailed.py

```diff
@@ -29,6 +32,8 @@ def _is_model_info_detailed(info: OrchestratorInfo) -> bool:
 
         def has_details(selection: Selection) -> bool:
             if isinstance(selection, InlineFragment):
+                if type_conditions and selection.type_condition not in type_conditions:
+                    return True
```

Comment:
Could we figure out a small unittest to hit this line?

Reply:
this fixes a problem of not getting any data with a detailed subscription when only asking for base properties, because the detail check becomes false and returns a `Subscription` instead of a `DetailedSubscription`.